packaging/nix/flake.nix

```
@@ -2,53 +2,43 @@
   description = "umu universal game launcher";
 
   inputs = {
-    nixpkgs = {
-      type = "github";
-      owner = "NixOS";
-      repo = "nixpkgs";
-      ref = "nixpkgs-unstable";
-    };
+    # FIXME: upstream package is in master, will be in nixos-unstable soon
+    nixpkgs.url = "github:nixos/nixpkgs/master";
```

When https://github.com/NixOS/nixpkgs/pull/369259 makes it to nixos-unstable ([tracker](https://nixpk.gs/pr-tracker.html?pr=369259)).

```suggestion
    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
```